app/views/movies/show.html.erb

```diff
@@ -0,0 +1,16 @@
+
+<h2> <%= @movie.title %> </h2>
+<p>Released in: <%=@movie.creation_year%></p>
+<p>Genre: <%=@movie.genre  %></p>
+<h4>Actors</h4>
+<p><% @movie.staff.each do |actor| %></p>
+    <p>Name:<%= actor.name %> </p>
+    <p>Age:<%= actor.age %></p>
+    <% end %>
+<p>"Average Age of Actors: <%=@movie.actors.average_age%></p>
+
+<%= form_with url: "/movies/#{@movie.id}", method: :post, local: true, data: { turbo: false } do |form| %>
```

Comment:
A more RESTful path here could be: POST "/movies/:id/actors" to indicate creating an association between these 2 resources